prismic-model/json/events.json

```diff
@@ -241,6 +241,12 @@
           }
         }
       }
+    },
+    "backgroundTexture" : {
+      "type" : "Image",
+      "config" : {
+        "label" : "Background texture"
+      }
```

Comment:
Should we not be making this a select?

Reply:
Or potentially another type, which has the background name, and we make this a select of that type?

Reply:
I wondered about that, but wasn't sure if it was really any better model-wise

Reply:
```
type Event {| id: string, title: string |};
type WebsiteEvent = {| ...Event, backgroundTexture: string |};
```

Reply:
Ta. https://xkcd.com/386/